### PR TITLE
LArSoft 09.90.01 build changes.

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by knoepfel** [comment]
@gartung, let me merge PR #2 first and then can you rebase this?

**Comment by knoepfel** [comment]
> @gartung, let me merge PR https://github.com/LArSoft/larsoft-spack-recipes/pull/2 first and then can you rebase this?

PR #2 now merged; @gartung, can you rebase?

File: packages/larrecodnn/package.py [hunk 1]
```diff
@@ -56,11 +56,15 @@ def cmake_args(self):
...
             self.define(
-                "DCMAKE_PREFIX_PATH",
+                "CMAKE_PREFIX_PATH",
                 "{0}/lib/python{1}/site-packages/torch".format(
                     self.spec["py-torch"].prefix, self.spec["python"].version.up_to(2)
                 ),
             ),
+            self.define(
+                "DELAUNATOR_INC",
+                "{0}/include".format(self.spec["delaunator-cpp"].prefix)
```
Comment (knoepfel) [inline packages/larrecodnn/package.py:66]: Suggest:

```diff
- "{0}/include".format(self.spec["delaunator-cpp"].prefix)
+ self.spec["delaunator-cpp"].prefix.include
```

File: packages/larwirecell/package.py [hunk 2]
```diff
@@ -61,12 +61,18 @@ def patch(self):
...
+            "jsoncpp jsonnet",
+            "Modules/FindWireCell.cmake",
+        )
 
     @cmake_preset
     def cmake_args(self):
         return [
             self.define_from_variant("CMAKE_CXX_STANDARD", "cxxstd"),
             self.define("IGNORE_ABSOLUTE_TRANSITIVE_DEPENDENCIES", True),
+            self.define("jsoncpp_DIR", "{0}".format(self.spec["jsoncpp"].prefix))
```
Comment (knoepfel) [inline packages/larwirecell/package.py:75]: Suggest:

```diff
- self.define("jsoncpp_DIR", "{0}".format(self.spec["jsoncpp"].prefix))
+ self.define("jsoncpp_DIR", self.spec["jsoncpp"].prefix)
```

File: packages/larreco/package.py [hunk 3]
```diff
@@ -50,6 +48,8 @@ class Larreco(CMakePackage, FnalGithubPackage):
     depends_on("rstartree")
     depends_on("tbb")
 
+    patch('09.25.00.patch', when='@09.25.00')
```
Comment (knoepfel) [inline packages/larreco/package.py:51]: Are you able to make a `larreco` PR with the changes in this patch?  If not, let me know and I will do it.

File: packages/larwirecell/package.py [hunk 4]
```diff
@@ -61,12 +61,18 @@ def patch(self):
             "find_package(Boost COMPONENTS graph date_time exception filesystem iostreams stacktrace_basic)\nfind_package(art ",
             "CMakeLists.txt",
         )
+        filter_file(
+            r"jsoncpp_lib jsonnet_lib",
+            "jsoncpp jsonnet",
+            "Modules/FindWireCell.cmake",
+        )
```
Comment (knoepfel) [inline packages/larwirecell/package.py:68]: Can you make a PR in `larwirecell` with these changes?  If not, let me know

**Review by knoepfel — COMMENTED** [review]
@gartung, these changes are excellent, Patrick.  Thanks!  I have a couple questions for you and a couple suggestions.

File: packages/larwirecell/package.py (hunk 4 above)
Comment (gartung) [reply packages/larwirecell/package.py:68]: I will make a pull request.

**Comment by gartung** [comment]
@knoepfel can I merge this?

**Pull request merged**